README.md

```diff
@@ -0,0 +1,94 @@
+Ecosystem Edge NVIDIA-CI - Golang Automation CI
+=======
+# NVIDIA-CI
+
+## Overview
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) is an automation/CI framework to test NVIDIA operators, starting with the GPU Operator.
+This project is based on golang + [ginkgo](https://onsi.github.io/ginkgo) framework.
+
+### Project requirements
+* golang v1.20.x
+* ginkgo v2.x
+
+## nvidia-ci
+The [nvidia-ci](https://github.com/rh-ecosystem-edge/nvidia-ci) project is designed to test a pre-installed OpenShift Container Platform (OCP) cluster which meets the following requirements:
+
+### Mandatory setup requirements:
+* OCP cluster installed with version >=4.12
+
+### Supported setups:
+* Regular cluster 3 master nodes (VMs or BMs) 2 workers (VMs or BMs)
```

Comment:
Aren't we supporting 3 workers instead of 2?

Reply:
This should read minimum 2 worker nodes and 3 master nodes.  So 3 worker nodes should be supported.

Reply:
Updated

Reply:
Seems like this one isn't resolved yet.